sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2764,6 +2764,9 @@ def expand(self, input):
 
   class ConvertToBeamRows(PTransform):
     def __init__(self, schema, dynamic_destinations):
+      if not isinstance(schema,
+                        (bigquery.TableSchema, bigquery.TableFieldSchema)):
+        schema = bigquery_tools.get_bq_tableschema(schema)
```

Comment:
do we need remove the lines in beam_row_from_dict?

Reply:
I could remove it but
beam_row_from_dict is publicly used method so removing this if would be potentially breaking change.